sustainml_cpp/src/cpp/common/Common.hpp

```diff
@@ -144,6 +145,25 @@ inline Topics get_topic_from_name(
     return output;
 }
 
+inline uint32_t parse_sustainml_env(
+        const uint32_t& option)
+{
+    uint32_t domain_to_use = option;
+    if (const char* env = std::getenv("SUSTAINML_DOMAIN_ID"))
```

Comment:
One more comment, pls move the "SUSTAINML_DOMAIN_ID" to be a 
`static constexpr const char* SUSTAINML_DOMAIN_URI = "SUSTAINML_DOMAIN_ID";`
defined in `Common.hpp` so that it can be easily modified or located at the same place with further env variables that we could add